webpack.config.js

```diff
@@ -52,10 +53,15 @@ module.exports = {
             loader: 'json-loader'
         }]
     },
-    postcss: [autoprefixer],
+    postcss: [
+        autoprefixer({
+            browsers: ['last 3 versions', 'Safari >= 8', 'iOS >= 8']
```

Comment:
This could be a discussion for outside this PR, but I have mixed feelings about putting `'last 3 versions'` in here. By my understanding, putting that here (and presumably the `webpack.config.js` for other modules) means that unless our various modules have the same version of `autoprefixer` they might target different browsers.

I guess that's "easy" to fix by making sure we standardize on a particular version of `autoprefixer`... is that the way to go in your opinion?